src/ui/command.rs

```diff
@@ -443,6 +443,14 @@ fn parse_command(msg: &str) -> Event {
             }
         }
         Some("/quit") | Some("/q") => Event::Quit,
+        Some("/lua") => {
+            let p = lc_msg[4..].trim();
+            if !p.is_empty() {
+                Event::RunLua(p.to_string())
+            } else {
+                Event::Info("USAGE: /lua <code>".to_string())
+            }
+        }
```

Comment:
Could you ad some help documentation for this command? Both in help.md and perhaps under scripting.md with a category 'command'

Reply:
Separate category doesn't really fit, since the text implies those are categories for the API on `blight`. I added a reference to the command where I thought appropriate, though,